lms/djangoapps/courseware/tests/test_self_paced_overrides.py

```diff
@@ -26,6 +26,7 @@ class SelfPacedDateOverrideTest(ModuleStoreTestCase):
     """
 
     def setUp(self):
+        self.reset_setting_cache_variables()
```

Comment:
The test may have always consistently passed before.  However, the introduction of a new test file in this PR, seemed to have changed the order in which the original test files were run.  This surfaced the underlying issue in the pre-existing tests.